.agents/skills/get-reviews/fetch_reviews.sh

```
@@ -0,0 +1,29 @@
+mkdir -p 'plans/$PR_NUMBER/fetched'
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)